dbms/src/Server/Server.cpp

```diff
@@ -606,13 +606,6 @@ int Server::main(const std::vector<std::string> & /*args*/)
         log);
 
     ProxyStateMachine proxy_machine{log, std::move(proxy_conf)};
-
-#ifdef USE_JEMALLOC
-    LOG_INFO(log, "Using Jemalloc for TiFlash");
-#else
-    LOG_INFO(log, "Not using Jemalloc for TiFlash");
-#endif
-
```

Comment:
these lines are useless because `setupAllocator` will output the corrsponding logging